v22.1/import.md

```diff
@@ -8,24 +8,28 @@ docs_area: reference.sql
 
 The `IMPORT` [statement](sql-statements.html) imports the following types of data into CockroachDB:
 
-- [Avro][avro]
-- [CSV/TSV][csv]
-- [Postgres dump files][postgres]
+- [PostgreSQL dump files][postgres]
 - [MySQL dump files][mysql]
-- [CockroachDB dump files](cockroach-dump.html)
-- [Delimited data files](#delimited-data-files)
 
-## Considerations
+To import CSV, Avro, or delimited data files, see [`IMPORT INTO`](import-into.html).
+
+{{site.data.alerts.callout_danger}}
+Certain `IMPORT TABLE` statements that defined the table schema inline are **not** supported in v22.1 and later versions. These include running `IMPORT TABLE ... CREATE USING` and `IMPORT TABLE` with any non-bundle format (`CSV`, `DELIMITED`, `PGCOPY`, or `AVRO`) data types.
 
-- `IMPORT` only works for creating new tables. For information on how to import into existing tables, see [`IMPORT INTO`](import-into.html). Also, for instructions and working examples on how to migrate data from other databases, see the [Migration Overview](migration-overview.html).
+To import data into a new table, use [`CREATE TABLE`](create-table.html) followed by [`IMPORT INTO`](import-into.html).
 
-{% include {{ page.version.version }}/import-table-deprecate.md %}
+`IMPORT INTO` supports CSV/TSV, Avro, and delimited data files. For an example, read [Import into a new table from a CSV file](import-into.html#import-into-a-new-table-from-a-csv-file).
+{{site.data.alerts.end}}
+
+## Considerations
 
 - `IMPORT` is a blocking statement. To run an import job asynchronously, use the [`DETACHED`](#options-detached) option.
 - `IMPORT` cannot be used within a [rolling upgrade](upgrade-cockroach-version.html).
-- `IMPORT` cannot directly import data to `REGIONAL BY ROW` tables that are part of [multi-region databases](multiregion-overview.html).  Instead, use [`IMPORT INTO`](import-into.html) which supports importing into `REGIONAL BY ROW` tables.
+- As of v22.1, `IMPORT TABLE ... CREATE USING` and `IMPORT TABLE` with `CSV`, `DELIMITED`, `PGCOPY`, or `AVRO` data types are **not** supported. Instead, use `CREATE TABLE` and `IMPORT INTO`; see this [example](import-into.html#import-into-a-new-table-from-a-csv-file) for more detail.
```

Comment:
> IMPORT TABLE` with `CSV`... data types are **not** supported.

While this is _correct_ I think it could be easy for a casual reader to read this incorrectly, and assume IMPORT of CSV, Avro, etc data is unsupported. It might be clearer if we shifted the focus to the creation aspect: 
> `IMPORT TABLE` statements, which supply a schema to create as well as data, in a format such as CSV or Avro, to load into the new table at the same time, is **no longer supported**; instead use CREATE TABLE blah blah.

Reply:
Hmm, yes good point. I'll pull down the language from the warning above the considerations section to have here as well, so there is no confusion in how this is described: 

> Certain `IMPORT TABLE` statements that defined the table schema inline are **not** supported in v22.1 and later versions. These include running `IMPORT TABLE ... CREATE USING` and `IMPORT TABLE` with any non-bundle format (`CSV`, `DELIMITED`, `PGCOPY`, or `AVRO`) data types. Instead, use `CREATE TABLE` and `IMPORT INTO`; see this [example](import-into.html#import-into-a-new-table-from-a-csv-file) for more detail.

I think it's a good idea to keep both instances warning + considerations to avoid users missing one or the other, or in the case of future page changes.